build.cake

```diff
@@ -766,12 +766,31 @@ Task("PackageZip")
         Zip(currentImageDir, File(ZIP_PACKAGE_SL), zipFiles);
     });
 
-Task("PackageMsi")
+
+Task("PackageRunnersMsi")
```

Comment:
I don't like the name, as we are only packaging one runner, the Console runner. We have already deprecated the NUnit.Runners nuget package in favor of NUnit.Console, so I don't think  we want to create another package we may subsequently deprecate.